docs/doxygen.json

```diff
@@ -1,3 +1,3 @@
 {
-  "indexPath": "../build/xml/index.xml"
+  "indexPath": "../.build/xml/index.xml"
```

Comment:
This is problematic. We are hardcoding a build folder name here.
It could be worked around by symlink or something like that from CMake.